docs/wip/traffic/permissive_traffic_policy_mode.md

```diff
@@ -0,0 +1,343 @@
+
+The alternative to permissive traffic policy mode is SMI traffic policy mode, where traffic between applications is denied by default and explicit SMI traffic policies are necessary to allow application connectivity. When policy enforcement is neccesary, SMI traffic policy mode must be used instead.
+
+## Configuring permissive traffic policy mode
+Permissive traffic policy mode can be enabled or disabled at the time of OSM install, or after OSM has been installed.
+
+### Enabling permissive traffic policy mode
+
+Enabling permissive traffic policy mode implicitly disables SMI traffic policy mode.
+
+During OSM install:
+```bash
+osm install --enable-permissive-traffic-policy=true
+```
+
+After OSM has been installed:
+```bash
+osm mesh upgrade --enable-permissive-traffic-policy=true
+```
+
+### Disabling permissive traffic policy mode
+
+Disabling permissive traffic policy mode implicitly enables SMI traffic policy mode.
+
+During OSM install:
+```bash
+osm install --enable-permissive-traffic-policy=false
+```
+
+After OSM has been installed:
+```bash
+osm mesh upgrade --enable-permissive-traffic-policy=false
+```
+
+## How does it work
+When permissive traffic policy mode is enabled, OSM controller discovers all services that are a part of the mesh and programs wildcard traffic routing rules on each Envoy proxy sidecar to reach every other service in the mesh. Additionally, each proxy fronting workloads that are associated with a service is configured to accept all traffic destined to the service. Depending on the application protocol of the service (HTTP, TCP, gRPC etc.), appropriate traffic routing rules are configured on the Envoy sidecar to allow all traffic for that particular type.
+
+## Sample demo
+
+### HTTP traffic in permissive mode
+
+The following demo shows an HTTP `curl` client making HTTP requests to the `httpbin` service using permissive traffic policy mode.
+
+1. Install OSM with permissive traffic policy mode enabled.
+    ```bash
+    osm install --enable-permissive-traffic-policy=true
+    ```
+
+1. Deploy the `httpbin` service into the `httpbin` namespace after enrolling its namespace to the mesh. The `httpbin` service runs on port `14001`.
+
+    ```bash
+    # Create the httpbin namespace
+    kubectl create namespace httpbin
+
+    # Deploy httpbin service in the httpbin namespace
+    kubectl apply -f docs/example/manifests/samples/httpbin/httpbin.yaml -n httpbin
+    ```
+
+    Confirm the `httpbin` service and pods are up and running.
+
+    ```console
+    $ kubectl get svc -n httpbin
+    NAME      TYPE        CLUSTER-IP     EXTERNAL-IP   PORT(S)     AGE
+    httpbin   ClusterIP   10.96.198.23   <none>        14001/TCP   20s
+    ```
+
+    ```console
+    $ kubectl get pods -n httpbin
+    NAME                     READY   STATUS    RESTARTS   AGE
+    httpbin-5b8b94b9-lt2vs   2/2     Running   0          20s
```

Comment:
Is there a step missing here (and for curl) to `osm namespace add` before creating the pods so they get a sidecar injected?

Reply:
You are right. Fixed this.